packages/client/hmi-client/src/components/workflow/tera-workflow-node.vue

```diff
@@ -21,20 +21,68 @@
 
 <script setup lang="ts">
 import { WorkflowNode } from '@/types/workflow';
-import { ref, onMounted } from 'vue';
+import { ref, computed, onMounted, onBeforeUnmount } from 'vue';
+
+const emit = defineEmits(['dragging']);
 
 const props = defineProps<{
 	node: WorkflowNode;
 }>();
 
-const nodeStyle = ref({
+const nodeStyle = computed(() => ({
 	minWidth: `${props.node.width}px`,
 	minHeight: `${props.node.height}px`,
 	top: `${props.node.y}px`,
 	left: `${props.node.x}px`
+}));
+
+const workflowNode = ref<HTMLElement>();
+
+let tempX = 0;
+let tempY = 0;
+let dragStart = false;
+
+const startDrag = (evt: MouseEvent) => {
+	console.log('start', evt.x, evt.y);
```

Comment:
Can we remove these logs

Reply:
I would like to keep the console.log around for a bit longer for debugging, will clean up with we have a full version-0 or version-1.